service-deployment/clusterconfig-example.yaml

```diff
@@ -85,6 +85,18 @@ clusterinfo:
     cluster_status_uri: http://10.0.3.9:9090
     # port for webportal
     server_port: 9286
+  
+  prometheusinfo:
+    # The address of the prometheus to connect to
+    prometheus_url: http://10.0.3.9:9090
+    # port for prometheus port
+    prometheus_port: 9090
```

Comment:
you don't need this. prometheus deployment handles this, right?

Reply:
It is used to replace the prom-datasource.yaml.template when building the grafana docker. I didn't include this file in the previous PR.